packages/@okta/vuepress-site/docs/guides/saml-application-setup/configure-saml-in-okta/index.md

```diff
@@ -3,22 +3,26 @@ title: Configure SAML in Okta
 ---
 To configure the SAML settings of your SAML Service Provider app in Okta:
 
-1.  On the **SAML Settings** page, paste the following test URL into the **Single sign on URL** and **Audience URI (SP Entity ID)** fields: `http://example.com/saml/sso/example-okta-com`.
+1. On the **SAML Settings** page, paste the ACS URL that you gathered in the <GuideLink link="../overview">first step</GuideLink> into the **Single sign on URL** field.
 
-2. In the **ATTRIBUTE STATEMENTS (OPTIONAL)** section, add three attribute statements:
+2. Paste the Audience URI that you gathered in the <GuideLink link="../overview">first step</GuideLink> into the **Audience URI (SP Entity ID)** field.
+
+    > **Note**: If you are just testing the setup and are using a SAML SP such as this [SAML Service Provider on GitHub](https://github.com/mcguinness/saml-sp), enter this test URL into the **Single sign on URL** and **Audience URI (SP Entity ID)** fields: `http://example.com/saml/sso/example-okta-com`.
+
+3. In the **ATTRIBUTE STATEMENTS (OPTIONAL)** section, add three attribute statements, and then any other required SAML attributes for your app:
```

Comment:
These three attributes are just example, so maybe it can be rephrased to
`In the **ATTRIBUTE STATEMENTS (OPTIONAL)** section, and the required SAML attributes for your app which you gathered in the <GuideLink link="../overview">first step</GuideLink>. For example:`